src/assets/images/index.jsx

```diff
@@ -0,0 +1,13 @@
+import Image from "next/image";
+
+export const HeroImage = (properties) => (
+  <Image
+    width={0}
+    height={0}
+    sizes="100vw"
+    style={{width: "100%", height: "auto"}}
```

Comment:
can we use tailwind classes here ?

Reply:
Tried using it,  but somehow it doesn't take effect,  that's why I need to check for other solutions for responsiveness or rather use html img tag instead of next Image component.